hedera-node/hedera-app/src/test/java/com/hedera/node/app/records/impl/codec/RunningHashesTranslatorTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.records.impl.codec;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.hedera.hapi.node.state.blockrecords.RunningHashes;
+import com.hedera.test.utils.TxnUtils;
+import com.swirlds.common.crypto.Hash;
+import com.swirlds.common.crypto.RunningHash;
+import java.io.IOException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class RunningHashesTranslatorTest {
+
+    private com.hedera.node.app.service.mono.stream.RecordsRunningHashLeaf subject;
+
+    @BeforeEach
+    void setUp() {
+        RunningHash runningHash = new RunningHash(new Hash(TxnUtils.randomUtf8Bytes(48)));
+        subject = new com.hedera.node.app.service.mono.stream.RecordsRunningHashLeaf(runningHash);
+    }
+
+    @Test
+    void createBlockInfoFromMerkleNetworkContext() throws IOException {
+
+        final RunningHashes runningHashes = RunningHashesTranslator.runningHashesFromRecordsRunningHashLeaf(subject);
+
+        assertTrue(runningHashes.runningHash().length() > 0);
+        assertTrue(runningHashes.nMinus1RunningHash().length() > 0);
+        assertTrue(runningHashes.nMinus2RunningHash().length() > 0);
+        assertTrue(runningHashes.nMinus3RunningHash().length() > 0);
```

Comment:
Can we have a more precise test that verifies the translated object actually has the right hash values, and not just are longer than 0?